examples/panel-datalinks/src/components/DataLinksPanel.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { PanelProps, getFieldDisplayValues } from '@grafana/data';
+import { SimpleOptions } from 'types';
+import { css, cx } from '@emotion/css';
+import { DataLinksContextMenu, useStyles2, useTheme2 } from '@grafana/ui';
+
+interface Props extends PanelProps<SimpleOptions> {}
+
+export const DataLinksPanel = ({ data, width, height, options, replaceVariables, fieldConfig, timeZone }: Props) => {
+  const styles = useStyles2(getStyles);
+  const theme = useTheme2();
+
+  const fieldDisplayValues = getFieldDisplayValues({
```

Comment:
I've struggled to find any documentation on how to work with datalinks so pieced things together from the examples I could find in core. Please let me know if this example isn't doing things in the preferred way.

Reply:
I think @dprokop is the expert on this area. But as far as I know this is the way to go. Great work on this @jackw!